scripts/certs/generate-certs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# SSL Certificate Generator for Local Testing
+# Creates self-signed certificates for use with Traefik
+# Supports localhost and custom domain testing
+
+# Usage Examples:
+# For localhost testing: ./generate-certs.sh
+# For custom domain testing: ./generate-certs.sh yourdomain.com
+
+# Output Files:
+# cert.key - SSL private key (KEEP SECURE)
+# cert.crt - SSL public certificate
+# cert.pem - Combined certificate and key (for import)
+
+# Set domain name
+DOMAIN_NAME=${1:-"localhost"}
+# Set cert directory
+CERT_DIR=${2:-"$HOME/.certs"}
+
+if [ ! -f "${CERT_DIR}/cert.key" ]; then
+
+  cd "${CERT_DIR}"
```

Comment:
```suggestion
  mkdir -p "${CERT_DIR}"
  cd "${CERT_DIR}"
```

suggestion to prevent a minor error for new users.

scripts/certs/generate-certs.sh: line 25: cd: /home/shson/.certs: No such file or directory
make: *** [Makefile:41: certs] Error 1